Sources/CreateAPI/Generator/Generator+Schemas.swift

```diff
@@ -337,7 +337,16 @@ extension Generator {
     }
 
     private func makeInlineProperties(for type: TypeName, object: JSONSchema.ObjectContext, context: Context) throws -> [Property] {
-        var keys = object.properties.keys
+        let excludedProperties = options.entities.exclude
+            .filter { $0.name == type.rawValue }
+            .compactMap { $0.property }
+        let excludedDiff = Set(excludedProperties).subtracting(object.properties.keys)
```

Comment:
```suggestion
        let unknownProperties = Set(excludedProperties).subtracting(object.properties.keys)
```

or something that's a bit of descriptive maybe?